frontend/packages/ceph-storage-plugin/src/components/ocs-install/install-wizard/configure.tsx

```diff
@@ -123,7 +126,7 @@ export const EncryptionFormGroup: React.FC<EncryptionFormGroupProps> = ({
         )}
         onChange={toggleEncryption}
       />
-      {encryptionChecked && (
+      {isKmsSupported && encryptionChecked && (
```

Comment:
AFAIK Cluster Wide encryption is supported in 4.6 as well.
```suggestion
      {encryptionChecked && (
```

Reply:
Yes this is for storage class based encryption.

Reply:
https://github.com/openshift/console/pull/7633/files#diff-ea96074271136809ad3a176f0fa5cee4f966328586398f3b4c9307cecf015de0R142
You're guarding this as well.

Reply:
Only https://github.com/openshift/console/pull/7633/files#diff-ea96074271136809ad3a176f0fa5cee4f966328586398f3b4c9307cecf015de0R152 this should be guarded.

Reply:
So, why to show checkbox when there is only one option ?

Reply:
![Screenshot from 2020-12-22 16-33-41](https://user-images.githubusercontent.com/25664409/103066082-24f4a080-45de-11eb-9396-226f48bd4162.png)

Reply:
no point in showing if only one option. But we need to show a checkbox for cluster-wide encryption.

Reply:
Then we need to show that checkbox as well. It would require css/implementation changes , since that box comes as a sub menu. 
Lets  keep it simple.

Reply:
@afreen23 but make sure the request is sent based on that because based on clusterwide encryption the flag is set in the request.

Reply:
We can make the Label change based on the flag. Enable Encryption can be Enable Cluster Wide Encryption.

Reply:
The idea to do it this way looks good as there is only one option.

Reply:
Yeah surely requests don't need any modification. They are good for encryption and rest of the features.